index.html

```diff
@@ -0,0 +1,476 @@
+            </article>
+            <article class="subwellArticle">
+                <a href="#" class="grid-link">
+                    <img src="images/subwell.png" alt="subwell">
+                    <p class="firstP">Subject Well</p>
+                    <p class="secondP">$ 10M</p>
+                    <p class="thirdP">SERIES A</p>
+                </a>
+                <a class="pharmaLink1" href="#"><i class="fa fa-tag" aria-hidden="true"></i> Pharmaceuticals, +2</a>
+                <a class="pharmaLink2" href="#"><i class="fa fa-suitcase" aria-hidden="true"></i> Windham Venture Partners, +2</a>
+                <a class="seriesALink"href="#">SubjectWell nabs $10 mln Series A <span>on PE Hub</span></a>
+            </article>
+            <article class="subwellArticle">
+                <a href="#" class="grid-link">
+                    <img src="images/subwell.png" alt="subwell">
+                    <p class="firstP">Subject Well</p>
+                    <p class="secondP">$ 10M</p>
+                    <p class="thirdP">SERIES A</p>
+                </a>
+                <a class="pharmaLink1" href="#"><i class="fa fa-tag" aria-hidden="true"></i> Pharmaceuticals, +2</a>
+                <a class="pharmaLink2" href="#"><i class="fa fa-suitcase" aria-hidden="true"></i> Windham Venture Partners, +2</a>
+                <a class="seriesALink"href="#">SubjectWell nabs $10 mln Series A <span>on PE Hub</span></a>
+            </article>
+            <article id="subwellLastArticle"class="subwellArticle">
+                <a href="#" class="grid-link">
+                    <img src="images/subwell.png" alt="subwell">
+                    <p class="firstP">Subject Well</p>
+                    <p class="secondP">$ 10M</p>
+                    <p class="thirdP">SERIES A</p>
+                </a>
+                <a class="pharmaLink1" href="#"><i class="fa fa-tag" aria-hidden="true"></i> Pharmaceuticals, +2</a>
+                <a class="pharmaLink2" href="#"><i class="fa fa-suitcase" aria-hidden="true"></i> Windham Venture Partners, +2</a>
+                <a class="seriesALink"href="#">SubjectWell nabs $10 mln Series A <span>on PE Hub</span></a>
+            </article>
+        </section>
+        <section class="subjects">
+            <article class="categories">
+                <h2><a href="#">Apps > </a></h2>
+                <div class="back-img">
+                    <div class="backGradient">
+                        <p class="back-img-p">MOST POPULAR</p>
+                        <a class="back-img-a1" href="#">How to use Google to find podcast episodes on a specific topic</a>
+                        <a class="back-img-a2" href="#">
+                            Ivan Mehta<span>- 3 hours ago</span></a>
+                    </div>
+                </div>
+                <div class="img-2">
+                    <figure>
+                        <img id="womanDancing" src="images/back-img-2.jpg" alt="woman dancing">
```

Comment:
#### `HINT`

Consider using a class instead. This is being used in eight different places and an `id` value is supposed to be unique across the whole page.